composer.json

```diff
@@ -21,12 +21,12 @@
         "php": ">=7.1",
         "doctrine/orm": "^2.5",
         "friendsofsymfony/jsrouting-bundle": "^1.6|^2.0",
-        "symfony/framework-bundle": "^3.4|^4.1",
-        "symfony/options-resolver": "^3.4|^4.1",
-        "symfony/property-access": "^3.4|^4.1",
-        "symfony/security": "^3.4|^4.1",
-        "symfony/translation": "^3.4|^4.1",
-        "twig/twig": "^2.9"
+        "symfony/framework-bundle": "^3.4|^4.1|^5.0",
+        "symfony/options-resolver": "^3.4|^4.1|^5.0",
+        "symfony/property-access": "^3.4|^4.1|^5.0",
+        "symfony/security": "^3.4|^4.1|^5.0",
```

Comment:
symfony/security has no 5.0 version
https://github.com/symfony/security/releases

Reply:
Ah ... strange ... I must've misread it before.
Just looking into this a little further, the only components in the code seem to be the HttpKernel & Console ... did you want me to narrow this list down a little further, or just fix up the version reference for the moment?